src/20_Components/amp-social-share.html

```diff
@@ -32,37 +40,53 @@
     `amp-social-share` with `type="facebook"` requires to specify the [facebook app id](https://developers.facebook.com/apps) via `data-attribution`.
   -->
   <div>
-    <amp-social-share type="twitter" width="60" height="44"> </amp-social-share>
-    <amp-social-share type="gplus" width="60" height="44"></amp-social-share>
-    <amp-social-share type="email" width="60" height="44"></amp-social-share>
-    <amp-social-share type="pinterest" width="60" height="44"></amp-social-share>
-    <amp-social-share type="linkedin" width="60" height="44"></amp-social-share>
-    <amp-social-share type="facebook" width="60" height="44" data-param-app_id="254325784911610"></amp-social-share>
+    <amp-social-share type="twitter"></amp-social-share>
+    <amp-social-share type="gplus"></amp-social-share>
+    <amp-social-share type="email"></amp-social-share>
+    <amp-social-share type="pinterest"></amp-social-share>
+    <amp-social-share type="linkedin"></amp-social-share>
+    <amp-social-share type="facebook" data-param-app_id="254325784911610"></amp-social-share>
   </div>
 
   <!-- #### Configuration -->
   <!--
     Embed the `amp-social-share` widget [choosing a type](https://www.ampproject.org/docs/reference/components/amp-social-share#types), then configure the actions.
 
-    * `text` is the text to include in the share
-    * `url` is the URL to share
-    * `attribution` is where the share is attributed to.
+    * `width`, default 60px.
+    * `height`, default 44px.
+    * `data-param-text` is the text to include in the share.
+    * `data-param-url` is the URL to share.
+    * `data-param-attribution` is where the share is attributed to.
+
```

Comment:
Can you please add another empty line here. Otherwise the markdown processor messes up the next paragraph.